disk_objectstore/cli.py

```diff
@@ -183,3 +184,73 @@ def optimize(
         container.clean_storage(vacuum=vacuum)
     size = sum(f.stat().st_size for f in dostore.path.glob("**/*") if f.is_file())
     click.echo(f"Final container size: {round(size/1000, 2)} Mb")
+
+
+@main.command("backup")
+@click.argument("dest", nargs=1, type=click.Path())
+@click.option(
+    "--keep",
+    default=1,
+    help="Number of previous backups to keep in the destination. (default: 1)",
+)
+@click.option(
+    "--rsync_exe",
+    default="rsync",
+    help="Specify the 'rsync' executable, if not in PATH. Used for both local and remote destinations.",
+)
+@click.option(
+    "--verbosity",
+    default="info",
+    help="Set verbosity [silent|info|debug], default is 'info'.",
+)
+@pass_dostore
+def backup(
+    dostore: ContainerContext, dest: str, keep: int, rsync_exe: str, verbosity: str
+):
+    """Create a backup of the container to destination location DEST, in a subfolder
+    backup_<timestamp>_<randstr> and point a symlink called `last-backup` to it.
```

Comment:
```suggestion
    """Create a backup of a container.
    
    The backup is created in at the `DEST` path, in a subfolder
    backup_<timestamp>_<randstr> and point a symlink called `last-backup` to it.
```

Reply:
Adapted in a separate commit to fix the text a bit.